### PR TITLE
Feature/r branch

### PR DESCRIPTION
**Pull request by Rusoch**
# Pull Request Template for ArtOa

## Description

Please provide a summary of the changes in this pull request:

-   What does this PR do?
-   Why are these changes necessary?
-   Any relevant links or references?

Fixes: [issue number/link]

---

## Type of Change

Please select the type of change this PR introduces (tick the appropriate box):

-   [ ] 🐛 Bug fix (non-breaking change that fixes an issue)
-   [ ] ✨ New feature (non-breaking change that adds functionality)
-   [ ] 🚨 Breaking change (fix or feature that would cause existing functionality
        to not work as expected)
-   [ ] 🔄 Refactor (code or file structure changes without affecting functionality)
-   [ ] 📝 Documentation update (changes or additions to documentation)
-   [ ] 🚀 Performance improvement (optimization in code or resources)

---

## Checklist

Please ensure the following tasks have been completed before requesting a review:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have run tests to ensure my changes work as expected
-   [ ] I have added/updated necessary tests or documentation (if applicable)
-   [ ] I have self-reviewed my code and refactored any areas that need improvement
-   [ ] There are no console errors or warnings related to my changes
-   [ ] The PR includes a clear, detailed title and description
-   [ ] Any relevant issues have been linked or mentioned in this PR
-   [ ] I have reviewed the acceptance criteria and met all requirements
-   [ ] I have considered edge cases and errors that may occur due to my changes

---

## Testing Instructions

Please describe how your changes can be tested:

1. **Test Setup:** (Any steps for environment setup)
2. **Test Steps:**
    - Step 1: ...
    - Step 2: ...
    - ...

---

## Screenshots / Visual Changes

Please attach any relevant screenshots, videos, or GIFs to illustrate the impact
of this change (if applicable).

---

## Additional Notes

Is there anything else the reviewer should know about this PR? Any known issues,
challenges, or decisions made during implementation?

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -61,7 +61,6 @@ of this change (if applicable).
 
 ---
 
-
```
Comment (Alsandre) [inline .github/pull_request_template.md:64]: This is redundant, it is caused by misplaced .prettierrc

File: .prettierrc.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
     "tabWidth": 4,
     "printWidth": 100,
-    "insertFinalNewline": true
```
Comment (Alsandre) [inline .prettierrc.json:4]: This line should stay as is.

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
 export default function Home() {
-    return <h1>hello</h1>;
+    return (
+        <h1>
+            hello
+            <Logo />
+            <WelcomeMessage />
+            <InstallPopup />
+        </h1>
+    );
```
Comment (Alsandre) [inline src/app/page.tsx:13]: Please cleanup this component, as I have incoming changes for it. Otherwise it will cause "merge conflict"
